uds/translator/configurable_translator.py

```
@@ -0,0 +1,1002 @@
+        """
+        response_on_event = self.services_mapping.get(RequestSID.ResponseOnEvent, None)
+        if response_on_event is None:
+            raise ValueError("ResponseOnEvent service is not defined in this Translator.")
+        conditional_response = response_on_event.response_structure[1].mapping.get(event, None)
+        if conditional_response is None or len(conditional_response) < 3:
+            return None
+        event_type_record = deepcopy(conditional_response[2])
+        event_type_record.name = f"{event_type_record.name}#{event_number}"
+        return event_type_record
+
+    def __get_event_type_record_09_continuation(self, event_number: int) -> None | ConditionalMappingDataRecord:
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)